test/InputOutput/unit_point.jl

```diff
@@ -0,0 +1,36 @@
+using Test
+import ClimaCore
+
+using ClimaComms
+const comms_ctx = ClimaComms.context(ClimaComms.CPUSingleThreaded())
```

Comment:
```suggestion
const comms_ctx = ClimaComms.context()
```
And let's add a cuda job for this, if it's not already on buildkite